webapp/views/wpt-results.js

```diff
@@ -823,10 +852,16 @@ class WPTResults extends WPTColors(WPTFlags(PathInfo(LoadingState(TestRunsUIBase
     }
   }
 
-  platformID({browser_name, browser_version, os_name, os_version}) {
+  platformID({ browser_name, browser_version, os_name, os_version }) {
     return `${browser_name}-${browser_version}-${os_name}-${os_version}`;
   }
 
+  canAmendMetadata(node, index, testRun) {
```

Comment:
We probably want a test for this.

Reply:
This method is incomplete because we need to add the ability to triage directory (wildcard) and subtest. Let me add a TODO and come back to this in the following PR